Remove mac-bigsur layout test expectations

achristensen07 opened the pull request:
#### 8514f361564d211bdccd939c7e72c90a10c44f1d
<pre>
Remove mac-bigsur layout test expectations
<a href="https://bugs.webkit.org/show_bug.cgi?id=259318">https://bugs.webkit.org/show_bug.cgi?id=259318</a>
rdar://112492959

Reviewed by Ryan Haddad.

It hasn&apos;t been used for many months now.

* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/anchor-element-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attrib-string-colors-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attribute-string-for-copy-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-1-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-2-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-3-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-4-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-5-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-with-style-1-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-with-style-2-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-for-typing-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-for-typing-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/basic-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/comment-cdata-section-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/font-size-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/font-style-variant-effect-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/font-weight-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/letter-spacing-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/text-decorations-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/vertical-align-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/fast/scrolling/latching/iframe-latch-small-deltas-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/imported/w3c/web-platform-tests/html/canvas/offscreen/path-objects/2d.path.stroke.scale2-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/imported/w3c/web-platform-tests/html/canvas/offscreen/path-objects/2d.path.stroke.scale2.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-stretch-properties-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/imported/w3c/web-platform-tests/media-source/mediasource-addsourcebuffer-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/TestExpectations: Removed.
* LayoutTests/platform/mac-bigsur-wk2/fast/dom/HTMLAnchorElement/anchor-file-blob-convert-to-download-async-delegate-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/fast/dom/HTMLAnchorElement/anchor-file-blob-convert-to-download-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/fast/events/contextmenu-lookup-action-for-image-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/ssl/applepay/ApplePayError-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/close-and-server-script-exception-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/close-code-and-reason-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/handshake-fail-by-no-cr-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/handshake-fail-by-null-char-in-status-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/handshake-ok-with-http-version-beyond-1_1-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/send-object-tostring-check-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/workers/close-code-and-reason-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/workers/service/serviceworker-websocket.https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/media-source/mediasource-changetype-play-implicit-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/media-source/mediasource-invalid-codec-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/notifications/idlharness.https.any.sharedworker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/webcodecs/videoFrame-createImageBitmap.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/webcodecs/videoFrame-drawImage.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/websockets/basic-auth.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/websockets/basic-auth.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/TestExpectations: Removed.
* LayoutTests/platform/mac-bigsur/accessibility/math-multiscript-attributes-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/css/continuationCrash-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/events/mouse-cursor-multiframecur-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/events/wheel/wheelevent-in-horizontal-scrollbar-in-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/events/wheel/wheelevent-in-vertical-scrollbar-in-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/button-sizes-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/control-restrict-line-height-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/input-appearance-spinbutton-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/input-button-sizes-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/number/number-appearance-spinbutton-disabled-readonly-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/select-list-box-with-height-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/select/optgroup-rendering-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/invalid/missing-end-tag-expected.png: Removed.
* LayoutTests/platform/mac-bigsur/fast/invalid/missing-end-tag-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/capitalize-boundaries-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-avoid-orphaned-word-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-character-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-first-word-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-limit-before-after-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-limit-lines-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-locale-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphens-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/international/Geeza-Pro-vertical-metrics-adjustment-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/khmer-lao-font-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/midword-break-after-breakable-char-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/tests/paymentrequest/payment-request-change-shipping-option.https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/tests/paymentrequest/updateWith-shippingOptions.https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/tests/security/xss-DENIED-xsl-external-entity-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/tests/security/xss-DENIED-xsl-external-entity-redirect-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/wpt/fetch/fetch-metadata-websocket-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/cookies/value/value-ctl-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/html/canvas/element/manual/wide-gamut-canvas/canvas-createImageBitmap-e_srgb-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/html/dom/documents/resource-metadata-management/document-cookie-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/ignored-properties-001-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/not-participating-to-parent-layout-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/padding-border-margin/border-002-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/padding-border-margin/margin-003-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/padding-border-margin/padding-002-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/html5-tree/dynamic-childlist-001-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/media-source/mediasource-addsourcebuffer-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/media-source/mediasource-invalid-codec-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/payment-request/payment-request-constructor.https.sub-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/payment-request/payment-request-shippingOption-attribute.https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/005_wss&amp;wpt_flags=https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/007_wss&amp;wpt_flags=https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-65K-data.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-binary-65K-arraybuffer.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-binary-65K-arraybuffer.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-binary-arraybuffer.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-binary-arraybuffer.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-data.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-data.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-paired-surrogates.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-paired-surrogates.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-unicode-data.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/bufferedAmount-unchanged-by-sync-xhr.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-arraybuffer-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-arraybuffer_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-blob-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-blob_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-getting-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-getting_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-large-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-large_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-unicode-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-unicode_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/inspector/css/get-system-fonts-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/mathml/presentation/mo-stretch-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/mathml/presentation/roots-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/mathml/radical-fallback-expected.png: Removed.
* LayoutTests/platform/mac-bigsur/mathml/radical-fallback-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/media/media-source/mediasource-addsourcebuffer-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/media/media-source/mediasource-invalid-codec-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/platform/mac/fast/loader/file-url-mimetypes-3-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/platform/mac/fast/text/international/Geeza-Pro-vertical-metrics-adjustment-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/svg/custom/use-invalid-pattern-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/svg/custom/use-invalid-style-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/svg/hixie/error/012-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/tables/mozilla/bugs/bug2479-3-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/tables/mozilla/bugs/bug26178-expected.txt: Removed.
* Tools/CISupport/build-webkit-org/steps.py:
(CompileWebKit.buildCommandKwargs):
* Tools/CISupport/build-webkit-org/steps_unittest.py:
(TestCompileWebKit.test_success):
(TestCompileWebKit.test_bigsur_timeout): Deleted.
* Tools/CISupport/build-webkit-org/wkbuild.py:
(_should_file_trigger_build):
* Tools/Scripts/bisect-builds:
* Tools/Scripts/webkitpy/port/factory.py:
(PortFactory.all_port_names):

Canonical link: <a href="https://commits.webkit.org/266148@main">https://commits.webkit.org/266148@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/625c2d95cfcfce129804b48b725d0d3ad17a887e

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/13027 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/13345 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/13676 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/14765 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/12418 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/13095 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/15851 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/13371 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/15112 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/13194 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/13886 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/11017 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/15218 "Built successfully") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/13120 "Passed tests") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/11174 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/11772 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/18833 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/12249 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/11938 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/15146 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/12413 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/10298 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/12910 "Passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/11681 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/15999 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/1475 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/12256 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

ryanhaddad commented on Tools/CISupport/build-webkit-org/steps_unittest.py:
```diff
@@ -413,12 +413,12 @@ def test_success(self):
 
     def test_bigsur_timeout(self):
```
I think we can remove this test entirely given your change to `steps.py`

achristensen07 replied:
That caused webkitpy tests to fail, but only on EWS.  I couldn't hit the problem locally (on Sonoma, admittedly)

achristensen07 committed:
Remove mac-bigsur layout test expectations
https://bugs.webkit.org/show_bug.cgi?id=259318
rdar://112492959

Reviewed by Ryan Haddad.

It hasn't been used for many months now.

* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/anchor-element-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attrib-string-colors-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attribute-string-for-copy-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-1-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-2-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-3-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-4-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-5-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-with-style-1-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-with-style-2-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-for-typing-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/attributed-string-for-typing-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/basic-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/comment-cdata-section-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/font-size-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/font-style-variant-effect-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/font-weight-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/letter-spacing-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/text-decorations-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/editing/mac/attributed-string/vertical-align-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/fast/scrolling/latching/iframe-latch-small-deltas-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/imported/w3c/web-platform-tests/html/canvas/offscreen/path-objects/2d.path.stroke.scale2-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/imported/w3c/web-platform-tests/html/canvas/offscreen/path-objects/2d.path.stroke.scale2.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-stretch-properties-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk1/imported/w3c/web-platform-tests/media-source/mediasource-addsourcebuffer-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/TestExpectations: Removed.
* LayoutTests/platform/mac-bigsur-wk2/fast/dom/HTMLAnchorElement/anchor-file-blob-convert-to-download-async-delegate-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/fast/dom/HTMLAnchorElement/anchor-file-blob-convert-to-download-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/fast/events/contextmenu-lookup-action-for-image-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/ssl/applepay/ApplePayError-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/close-and-server-script-exception-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/close-code-and-reason-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/handshake-fail-by-no-cr-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/handshake-fail-by-null-char-in-status-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/handshake-ok-with-http-version-beyond-1_1-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/send-object-tostring-check-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/websocket/tests/hybi/workers/close-code-and-reason-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/http/tests/workers/service/serviceworker-websocket.https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/media-source/mediasource-changetype-play-implicit-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/media-source/mediasource-invalid-codec-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/notifications/idlharness.https.any.sharedworker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/webcodecs/videoFrame-createImageBitmap.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/webcodecs/videoFrame-drawImage.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/websockets/basic-auth.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur-wk2/imported/w3c/web-platform-tests/websockets/basic-auth.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/TestExpectations: Removed.
* LayoutTests/platform/mac-bigsur/accessibility/math-multiscript-attributes-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/css/continuationCrash-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/events/mouse-cursor-multiframecur-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/events/wheel/wheelevent-in-horizontal-scrollbar-in-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/events/wheel/wheelevent-in-vertical-scrollbar-in-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/button-sizes-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/control-restrict-line-height-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/input-appearance-spinbutton-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/input-button-sizes-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/number/number-appearance-spinbutton-disabled-readonly-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/select-list-box-with-height-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/forms/select/optgroup-rendering-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/invalid/missing-end-tag-expected.png: Removed.
* LayoutTests/platform/mac-bigsur/fast/invalid/missing-end-tag-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/capitalize-boundaries-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-avoid-orphaned-word-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-character-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-first-word-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-limit-before-after-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-limit-lines-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphenate-locale-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/hyphens-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/international/Geeza-Pro-vertical-metrics-adjustment-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/khmer-lao-font-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/fast/text/midword-break-after-breakable-char-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/tests/paymentrequest/payment-request-change-shipping-option.https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/tests/paymentrequest/updateWith-shippingOptions.https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/tests/security/xss-DENIED-xsl-external-entity-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/tests/security/xss-DENIED-xsl-external-entity-redirect-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/http/wpt/fetch/fetch-metadata-websocket-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/cookies/value/value-ctl-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/html/canvas/element/manual/wide-gamut-canvas/canvas-createImageBitmap-e_srgb-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/html/dom/documents/resource-metadata-management/document-cookie-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/ignored-properties-001-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/not-participating-to-parent-layout-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/padding-border-margin/border-002-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/padding-border-margin/margin-003-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/css-styling/padding-border-margin/padding-002-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/mathml/relations/html5-tree/dynamic-childlist-001-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/media-source/mediasource-addsourcebuffer-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/media-source/mediasource-invalid-codec-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/payment-request/payment-request-constructor.https.sub-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/payment-request/payment-request-shippingOption-attribute.https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/005_wss&wpt_flags=https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/007_wss&wpt_flags=https-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-65K-data.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-binary-65K-arraybuffer.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-binary-65K-arraybuffer.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-binary-arraybuffer.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-binary-arraybuffer.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-data.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-data.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-paired-surrogates.any-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-paired-surrogates.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/Send-unicode-data.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/bufferedAmount-unchanged-by-sync-xhr.any.worker-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-arraybuffer-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-arraybuffer_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-blob-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-blob_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-getting-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-getting_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-large-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-large_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-unicode-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-unicode_wss-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/inspector/css/get-system-fonts-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/mathml/presentation/mo-stretch-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/mathml/presentation/roots-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/mathml/radical-fallback-expected.png: Removed.
* LayoutTests/platform/mac-bigsur/mathml/radical-fallback-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/media/media-source/mediasource-addsourcebuffer-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/media/media-source/mediasource-invalid-codec-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/platform/mac/fast/loader/file-url-mimetypes-3-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/platform/mac/fast/text/international/Geeza-Pro-vertical-metrics-adjustment-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/svg/custom/use-invalid-pattern-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/svg/custom/use-invalid-style-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/svg/hixie/error/012-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/tables/mozilla/bugs/bug2479-3-expected.txt: Removed.
* LayoutTests/platform/mac-bigsur/tables/mozilla/bugs/bug26178-expected.txt: Removed.
* Tools/CISupport/build-webkit-org/steps.py:
(CompileWebKit.buildCommandKwargs):
* Tools/CISupport/build-webkit-org/steps_unittest.py:
(TestCompileWebKit.test_success):
(TestCompileWebKit.test_bigsur_timeout): Deleted.
* Tools/CISupport/build-webkit-org/wkbuild.py:
(_should_file_trigger_build):
* Tools/Scripts/bisect-builds:
* Tools/Scripts/webkitpy/port/factory.py:
(PortFactory.all_port_names):

Canonical link: https://commits.webkit.org/266148@main
